llama_stack/providers/adapters/safety/together/safety.py

```diff
@@ -0,0 +1,74 @@
+# Copyright (c) Meta Platforms, Inc. and affiliates.
+# All rights reserved.
+#
+# This source code is licensed under the terms described in the LICENSE file in
+# the root directory of this source tree.
+import pydantic
+from together import Together
+
+import asyncio
+
+from llama_stack.distribution.request_headers import get_request_provider_data
+from .config import TogetherSafetyConfig
+from llama_stack.apis.safety import *
+import logging
+
+class TogetherHeaderInfo(BaseModel):
+    together_api_key: str
+
+
+class TogetherSafetyImpl(Safety):
+    def __init__(self, config: TogetherSafetyConfig) -> None:
+        self.config = config
+
+    async def initialize(self) -> None:
+        pass
+
+    async def run_shield(
+            self, shield_type: str, messages: List[Message], params: Dict[str, Any] = None
```

Comment:
hey this still does not look properly formatted -- were you able to get `pre-commit` to run?

Reply:
yes, ran the pre-commit. Let me try it again